### ISSUE TITLE
number of steps for training SSD inception on CoCo

### ISSUE THREAD

**Issue by YknZhu** [tensorflow/models#5724]
Sorry to hijack this post
Could you please tell me how many steps are required to train COCO dataset on SSD Inception V2 model? I have checked tensorflow object detection api paper, it is mentioned LR rate was decreased by 0.95 for every 800k steps. But what is the total number of steps required to train COCO dataset? 
Please help me.
Thank you.

_Originally posted by @jillelajitta in https://github.com/tensorflow/models/issues/5662#issuecomment-437139417_

**Comment by YknZhu** [comment]
@jillelajitta usually we just train until your eval curve converges.

By inspecting the model checkpoint at https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md,  seems the released checkpoint is at step 5M steps.

**Comment by jillelajitta** [comment]
Thank you @YknZhu.
I parsed out 90 classes from OPEN IMAGE DATASET. I roughly have 600k train images and 200k val images. I want to train this dataset on coco pretrained inception v2 model. How many steps do you suggest me to train? What is the best learning rate to start with.

**Comment by YknZhu** [comment]
No much experience on open image but probably start from smaller learning rates and train until your model performance converges.

**Comment by jillelajitta** [comment]
Thank you. Do you have any idea how many steps to train approximately on pre trained coco model?
Train images :700k
val images: 300k

**Comment by jillelajitta** [comment]
Hi @YknZhu,
I'm using model_main.py for training. This file is evaluating for every 1200 steps but I want to evaluate for every 50k steps. I tried changing config file but no success. Could you please help me how to evaluate the model for every 50k steps?
Thanks.

**Comment by jillelajitta** [comment]
Hi @YknZhu,
I'm using model_main.py for training. This file is evaluating for every 1200 steps but I want to evaluate for every 50k steps. I tried changing config file but no success. Could you please help me how to evaluate the model for every 50k steps?
Please help me with this.
Thanks.

**Comment by AloneGu** [comment]
> @jillelajitta usually we just train until your eval curve converges.
> 
> By inspecting the model checkpoint at https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md, seems the released checkpoint is at step 5M steps.

Hi, I just want to ask how do you inspecting the model checkpoint to get [5M].

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by snake-arch** [comment]
@jillelajitta Hi, Were you able to solve the problem?. I am facing a similar problem and I was wondering if anyone could help me.
